app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author
+  belongs_to :post
+
+  def update_like_posts_counter
+    post.update(likes_counter: post.likess.count)
```

Comment:
- [ ] it seems you might have made a typo in this line because you typed 'likess' with double "ss"